qemu/tests/cfg/win_irq_check.cfg

```diff
@@ -0,0 +1,21 @@
+- win_irq_check:
+    type = win_irq_check
+    only Windows
```

Comment:
I think this line should be first line of params.

Reply:
moved the position.